gcloud-java-storage/src/main/java/com/google/gcloud/storage/HttpMethod.java

```diff
@@ -20,5 +20,5 @@
  * Http method supported by Storage service.
  */
 public enum HttpMethod {
-  GET, HEAD, PUT, POST, DELETE
+  GET, HEAD, PUT, POST, DELETE, OPTIONS
```

Comment:
Interesting. I am not sure where I got the original list though I think it is from [here](https://cloud.google.com/storage/docs/xml-api/get-bucket-cors?hl=en).
Maybe we should put all possible methods based on the [RFC](https://tools.ietf.org/html/rfc2616#section-5.1.1), which is referenced from [here](http://www.w3.org/TR/cors/#http-access-control-request-method)?